### PR TITLE
ffmpeg: add Nepali translation

### PR DESCRIPTION
**Pull request by psycho-arna**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ne/common/ffmpeg.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- अंकित छविहरूलाइ(`फ्रेम_1.jpg`, `फ्रेम_2.jpg`, आदि) भिडियो वा GIF मा जोड्नुहोस:
+
+`ffmpeg -i {{फ्रेम_%d.jpg}} -f image2 {{भिडियो.mpg|भिडियो.gif}}`
+
+- भिडियोको mm:ss बाट एकल फ्रेम निकाल्नुहोस् र यसलाई 128x128 रिजोल्युसनको छविको रूपमा सेभ गर्नुहोस् :
+
+`ffmpeg -ss {{mm:ss}} -i {{भिडियो.mp4}} -frames 1 -s {{128x128}} -f image2 {{छवि.png}}`
+
+- दिइएको सुरु समय mm:ss देखि अन्त्यसमय mm2:ss2 सम्म भिडियोलाइ काट्नुहोस (अन्त्य सम्म नै  काट्नलाई  -to फ्ल्याग हटाउनुहोस्):
```
Comment (navarroaxel) [inline pages.ne/common/ffmpeg.md:22]: Why 2 spaces here?
```suggestion
- दिइएको सुरु समय mm:ss देखि अन्त्यसमय mm2:ss2 सम्म भिडियोलाइ काट्नुहोस (अन्त्य सम्म नै काट्नलाई -to फ्ल्याग हटाउनुहोस्):
```

File: pages.ne/common/ffmpeg.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`ffmpeg -ss {{mm:ss}} -to {{mm2:ss2}} -i {{भिडियो.mp4}} -codec copy {{आउटपुट.mp4}}`
+
+- AVI भिडियोलाई MP4 मा रूपान्तरण गर्नुहोस्। अडियोको 128kbit बिटरेट राखेर AAC मा, भिडियोको CRF 23 राखेर h264 मा :
```
Comment (navarroaxel) [inline pages.ne/common/ffmpeg.md:26]:
```suggestion
- AVI भिडियोलाई MP4 मा रूपान्तरण गर्नुहोस्। अडियोको 128kbit बिटरेट राखेर AAC मा, भिडियोको CRF 23 राखेर h264 मा:
```

File: pages.ne/common/ffmpeg.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`ffmpeg -i {{इनपुट_भिडियो}}.avi -codec:audio aac -b:audio 128k -codec:video libx264 -crf 23 {{आउटपुट_भिडियो}}.mp4`
+
+- MKV भिडियोको अडियो वा भिडियो स्ट्रिमहरू पुन: एन्कोडिङ नगरी हानिरहित ढाँचामा MP4 मा बदल्नुहोस :
```
Comment (navarroaxel) [inline pages.ne/common/ffmpeg.md:30]:
```suggestion
- MKV भिडियोको अडियो वा भिडियो स्ट्रिमहरू पुन: एन्कोडिङ नगरी हानिरहित ढाँचामा MP4 मा बदल्नुहोस:
```

File: pages.ne/common/ffmpeg.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`ffmpeg -i {{इनपुट_भिडियो}}.mkv -codec copy {{आउटपुट_भिडियो}}.mp4`
+
+- MP4 भिडियोलाई VP9 कोडेकमा रूपान्तरण गर्नुहोस्। उत्कृष्ट गुणस्तरको लागि, CRF फ्ल्यागको प्रयोग गर्नुहोस् (सिफारिस गरिएको दायरा 15-35) र -b:video 0 पनि प्रयोग गर्नुहोस् :
```
Comment (navarroaxel) [inline pages.ne/common/ffmpeg.md:34]:
```suggestion
- MP4 भिडियोलाई VP9 कोडेकमा रूपान्तरण गर्नुहोस्। उत्कृष्ट गुणस्तरको लागि, CRF फ्ल्यागको प्रयोग गर्नुहोस् (सिफारिस गरिएको दायरा 15-35) र -b:video 0 पनि प्रयोग गर्नुहोस्:
```

File: pages.ne/common/ffmpeg.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- भिडियोको उचाइ 1000px मा स्केल गर्दै र फ्रेमरेटलाई 15 राखेर GIF को रूपमा सेभ गर्नुहोस :
+
+`ffmpeg -i {{भिडियो.mp4}} -vf 'scale=-1:{{1000}}' -r {{15}} {{आउटपुट.gif}}`
+
+- अंकित छविहरूलाइ(`फ्रेम_1.jpg`, `फ्रेम_2.jpg`, आदि) भिडियो वा GIF मा जोड्नुहोस:
+
+`ffmpeg -i {{फ्रेम_%d.jpg}} -f image2 {{भिडियो.mpg|भिडियो.gif}}`
+
+- भिडियोको mm:ss बाट एकल फ्रेम निकाल्नुहोस् र यसलाई 128x128 रिजोल्युसनको छविको रूपमा सेभ गर्नुहोस् :
```
Comment (navarroaxel) [inline pages.ne/common/ffmpeg.md:18]:
```suggestion
- भिडियोको mm:ss बाट एकल फ्रेम निकाल्नुहोस् र यसलाई 128x128 रिजोल्युसनको छविको रूपमा सेभ गर्नुहोस्:
```

File: pages.ne/common/ffmpeg.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# ffmpeg
+
+> भिडियो रूपान्तरण उपकरण।
+> थप जानकारी: <https://ffmpeg.org>.
+
+- भिडियोबाट ध्वनि निकाल्नुहोस् र MP3 को रूपमा सेभ गर्नुहोस:
+
+`ffmpeg -i {{भिडियो.mp4}} -vn {{ध्वनि}}.mp3`
+
+- भिडियोको उचाइ 1000px मा स्केल गर्दै र फ्रेमरेटलाई 15 राखेर GIF को रूपमा सेभ गर्नुहोस :
```
Comment (navarroaxel) [inline pages.ne/common/ffmpeg.md:10]:
```suggestion
- भिडियोको उचाइ 1000px मा स्केल गर्दै र फ्रेमरेटलाई 15 राखेर GIF को रूपमा सेभ गर्नुहोस:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**